job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PlotPluginContext.groovy

```diff
@@ -0,0 +1,19 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.ContextHelper
+import javaposse.jobdsl.dsl.Context
+
+class PlotPluginContext implements Context {
+    List<PlotPluginPlotContext> plots = []
+
+    void line(String dataStore, String group, Closure plotPluginPlotClosure) {
```

Comment:
All plot styles have the same options, use separate method to set the style, e.g. `style(String style)`. The dataStore option is not visible in the UI, so why must this be specified in the DSL?

In the UI, the group is a mandatory option, so it should be a method parameter, e.g.

```groovy
plot(String group) {
    style(String style)
    ...
}
```